<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gLabels - Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Locale</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select locale specific behavior.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Units</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="4" column="0">
           <widget class="QRadioButton" name="unitsPointsRadio">
            <property name="text">
             <string>Points</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="unitsCentimetersRadio">
            <property name="text">
             <string>Centimeters</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="unitsMillimetersRadio">
            <property name="text">
             <string>Millimeters</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="unitsInchesRadio">
            <property name="text">
             <string>Inches</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QRadioButton" name="unitsPicasRadio">
            <property name="text">
             <string>Picas</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unitsPointsRadio</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onUnitsRadiosChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unitsInchesRadio</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onUnitsRadiosChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unitsMillimetersRadio</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onUnitsRadiosChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unitsCentimetersRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onUnitsRadiosChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unitsPicasRadio</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onUnitsRadiosChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onUnitsRadiosChanged()</slot>
  <slot>onPreferedPaperSizesRadiosChanged()</slot>
 </slots>
</ui>
